### PR TITLE
feat(kit): new `maskitoPrefixPostprocessorGenerator`

### PR DESCRIPTION
**Pull request by nsbarsukov**
## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [X] Feature
- [ ] Refactoring
- [ ] Code style update
- [ ] Build or CI related changes
- [ ] Documentation content changes

## What is the current behavior?

Partially solves #231

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

File: projects/demo/src/pages/recipes/prefix/examples/1-pattern-mask/component.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import {ChangeDetectionStrategy, Component} from '@angular/core';
+
+import mask from './mask';
+
+@Component({
+    selector: 'prefix-doc-example-1',
+    template: `
+        <tui-input
+            ngModel=""
```
Comment (waterplea) [inline projects/demo/src/pages/recipes/prefix/examples/1-pattern-mask/component.ts:9]: Maybe we should add proper ngModel binding for the example?

File: projects/demo/src/pages/recipes/prefix/examples/1-pattern-mask/mask.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import {MaskitoOptions} from '@maskito/core';
+
+export default {
+    mask: ({value}) => {
+        const digitsCount = value.replace(/\D/g, '').length;
+
+        if (!digitsCount) {
```
Comment (waterplea) [inline projects/demo/src/pages/recipes/prefix/examples/1-pattern-mask/mask.ts:7]: Ternary FTW )

**Review by waterplea — APPROVED** [review]

**Review by vladimirpotekhin — APPROVED** [review]

File: projects/demo/src/pages/recipes/prefix/examples/1-pattern-mask/component.ts (hunk 1 above)
Comment (nsbarsukov) [reply projects/demo/src/pages/recipes/prefix/examples/1-pattern-mask/component.ts]: fixed

File: projects/demo/src/pages/recipes/prefix/examples/1-pattern-mask/mask.ts (hunk 2 above)
Comment (nsbarsukov) [reply projects/demo/src/pages/recipes/prefix/examples/1-pattern-mask/mask.ts]: changed to:
```ts
const digitsCount = value.replace(/\D/g, '').length;

return ['$', ...new Array(digitsCount || 1).fill(/\d/)];
```

**Pull request merged**